cypress/integration/twilio-video.spec.js

```diff
@@ -22,7 +22,8 @@ const caseRef = uuid();
  context('Startup', () => {
     
       before(() => { 
-                  cy.tabulaLogin(conferenceUrlPath,Cypress.env('login_user_name'),Cypress.env('login_user_password'));
+           
+                  cy.tabulaLogin(conferenceUrlPath,Cypress.env('loginAdminUserName'),Cypress.env('loginAdminPassword'));
```

Comment:
There is login function developed by Jonnie (I think). I want to make sure you and Jonnie are using the same function.
@Jonniedev @giladadon1997

Reply:
I suggest you and @Jonniedev will use the same function to login (the name of the function should not include TABula as this is not only TABula). Please MOVE this function to https://github.com/eScribers/authentication

We should have either one function or multiple functions for (Guest - w/o password, Registered user - with password ...) I think we should use one function with the following parameters
Language
UserIdentifier
Password
CaseReference